### DISCUSSION TITLE
Paypal or German Bank Account payments

### DISCUSSION THREAD

**Discussion by Bravokilo1980** [aclap-dev/video-downloadhelper#877] [Help and Q&A]
### Your question

Where is the option for Paypal or for German Bank accounts to buy your product? I couldn't find any.

### Addon & CoApp version and details

Version
    9.0.2.3

**Comment by jcfrog** [comment]
Hi,
Sorry we do not use paypal, you have to use Stripe for the payment. Stripe might offer some alternatives as google pay or apple pay.

**Reply by stereotron** [reply]
> Hi, Sorry we do not use paypal, you have to use Stripe for the payment. Stripe might offer some alternatives as google pay or apple pay.

Stripe seems to be the platform you are using for receiving money. Google Pay again wants my credit card number, which I don't have nor want to have. You relinquish money from European users as EC cards are common here. Credit cards cost an additional, annual fee. I'd really like to buy the premium version, yet I won't subscribe to a credit card company just for that.

**Reply by Darnulf** [reply]
Paypal or Klarna should be an option for payment to allow liscense purchase without a credit card
unless you don't want those people to buy a liscense.

**Reply by stereotron** [reply]
I'm afraid this isn't an option here anymore. But I found a way to get a [temporary credit card](https://github.com/aclap-dev/video-downloadhelper/discussions/1107#discussioncomment-10990124). 
In case you are an unlucky German Sparkassenkunde, they seem to exchange expired EC cards to EC cards with VISA debit function without an extra fee.
